src/protos/io/defang/v1/fabric.proto

```diff
@@ -690,3 +693,25 @@ message WhoAmIResponse {
   SubscriptionTier tier = 5;
   bool training_opt_out = 6;
 }
+
+message EstimateRequest {
+  string provider = 1;
+  string region = 2;
+  bytes pulumi_preview = 3;
```

Comment:
not a string?

Reply:
When should one prefer a string over a byte array?

Reply:
If it’s known to be UTF-8, use string

Reply:
When we pass defang outputs as JSON, we used a byte array, and when we pass compose files, we use a byte array. Should those be strings as well?